apps/users/api/internal/token_refresh.py

```
@@ -13,63 +13,62 @@
+            if getattr(settings, "SIMPLE_JWT", {}).get("ROTATE_REFRESH_TOKENS", False):
+                try:
+                    refresh.blacklist()
+                    # Get user from token payload
+                    user_id = refresh.payload.get("user_id")
+                    user = User.objects.get(id=user_id)
+                    new_refresh = RefreshToken.for_user(user)
+                    response_data["refresh"] = str(new_refresh)
+                except Exception as e:
+                    raise ItqanError(
+                        error_name="token_rotation_failed",
+                        message=f"Failed to rotate refresh token: {str(e)}",
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)